app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -561,6 +564,14 @@ function AcuantCapture(
       setAcuantFailureCookie(null);
 
       onCameraAccessDeclined();
+
+      // Due to a bug with Safari on iOS we force the page to refresh on the third
+      // time a user denies permissions.
+      onFailedCameraPermissionAttempt();
```

Comment:
I posted this in Slack, but realized I probably should have posted it here instead (with a couple of edits for clarity).

I've tested this branch out locally, and I see something that I'm not sure is intended behavior, or perhaps I'm misinterpreting text. It does look like this behavior is in the video provided, so it's probably me missing something.

With "after permissions being denied three times", I expected the page refresh to happen after I hit cancel on the the third try. But really, after I hit the "front of your license" button area 3x, saw the error message, and then I hit the "front of your license" button area for a 4th time, that is when it refreshes.

It feels a little jarring to me for that to be the timing, but maybe that's just me and this is behavior as intended.

Reply:
Thanks for mentioning this, it is jarring. I think we want to proceed as is.
- This should not be happening for too many folks.
- This is a mitigation, we aren't able to truly fix the issue, so we're trying to unblock users which this achieves.
It would be nice to revisit this at a later date and clean it up, but for now I think "users don't get stuck" is achieved in this PR.